libs/core/dtos/anime.dto.ts

```diff
@@ -0,0 +1,63 @@
+import { DateRangeDto } from './date-range.dto';
+
+/** Anime data dto. */
```

Comment:
_data_ doesn't provide much meaning here
```suggestion
/** Anime DTO. */
```